app/src/main/java/com/concordium/wallet/data/backend/wert/WertRepository.kt

```diff
@@ -0,0 +1,27 @@
+package com.concordium.wallet.data.backend.wert
+
+import com.concordium.wallet.App
+
+class WertRepository {
+
+    private val backend = App.appCore.getWertBackend()
+
+    suspend fun getWertSessionDetails(
```

Comment:
Why mixing suspend and callbacks? It can just throw an exception and have `WertSessionResponse` return type